onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -124,10 +129,142 @@ def _parse_line(line):
 
     return (scalefacs, zpoints)
 
+# Adding ability to record intermediate graph outputs
+selected_types = ['Conv', 'MatMul'] # node types to extend
+
+def augment_graph(model):
+    '''
+    Adds ReduceMin and ReduceMax nodes to all Conv and MatMul nodes in
+    model and ensures their outputs are stored as part of the graph output
+        parameter model: loaded FP32 ONNX model to quantize
+        return: augmented ONNX model
+    '''
+    added_nodes = []
+    added_outputs = []
+    for node in model.graph.node:
+        if node.op_type in selected_types:
+            input_name = node.name
```

Comment:
don't use the node name ... you should use the output name... this wont work,,, right now it works for you because you are explicitly adding ":0" to this name which is how your current mobilnet model is... but this is not scalable.... 

first check there is only 1 output for this node... (since we cant handle modes with multiple outputs right now)
then use the output[0] to get the name

Reply:
Fixed.